packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -107,20 +107,36 @@ export class AtomicSegmentedFacetScrollable implements InitializableComponent {
 
   private renderArrowClass(direction: ArrowDirection) {
     if (direction === 'left') {
-      return 'left-0 ' + (this.hideLeftArrow ? 'invisible opacity-0' : '');
+      return (
+        'left-0 ' +
+        (this.hideLeftArrow || !this.searchStatusState.firstSearchExecuted
```

Comment:
I had to check for `!this.searchStatusState.firstSearchExecuted` to make sure the arrows don't load before the segmented facets.

Reply:
could you put this in the render method instead for example

```typescript
render() {
    if (this.searchStatus.state.hasError) {
      return <Hidden></Hidden>;
    }
    return (
      <div part="scrollable-container" class="flex relative">
        {this.searchStatusState.firstSearchExecuted && this.renderArrow('left')}
        <div
          part="horizontal-scroll"
          class="wrapper-segmented flex flex-row overflow-x-scroll scroll-smooth"
          ref={(el) => (this.horizontalScroll = el as HTMLDivElement)}
        >
          <slot></slot>
        </div>
        {this.searchStatusState.firstSearchExecuted && this.renderArrow('right')}
      </div>
    );
}
```